libs/@local/hash-graphql-shared/LICENSE.md

```diff
@@ -0,0 +1,607 @@
+GNU Affero General Public License
```

Comment:
@nonparibus do we want AGPL here as well? I copied the same license as the one from the package I hoisted these from.

Reply:
Yes, this is correct. Thank you.